src_docs/developer/07.MannerOfThePullRequest.md

```diff
@@ -0,0 +1,85 @@
+# Manner of the Pull Request
+
+## Making Effective Comments on Pull Requests
+
+Pull Requests (PRs) are not just about submitting code — they are also a space for discussion and collaboration. Writing
+clear, relevant, and respectful comments is essential to keep development productive and enjoyable for everyone.
+
+This section is especially helpful for project managers, testers, and new contributors who want to provide feedback but
+are unsure how to do it effectively.
+
+---
+
+### 🎯 Stay Focused on the Scope
+
+Each PR typically aims to fix a specific problem or introduce a clearly defined feature. When reviewing a PR, please:
+
+- **Read the PR description** carefully. It explains the purpose and scope.
+- **Comment only on things directly related** to that purpose.
+- If you discover an unrelated issue, note it down elsewhere (e.g., in a new ticket or in a chat), rather than posting
+  it in the PR.
```

Comment:
```suggestion
- If you discover an unrelated issue, note it down elsewhere (e.g., in a new ticket or in a chat), rather than posting it in the PR.
```